### PR TITLE
Create Section/chapter 6 opening

### PR DESCRIPTION
**Pull request by YoyoyoChang**
Create Section/chapter 6 opening

### CODE REVIEW

File: game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        tg.immediate(context => {
+            const centaur: CDOTA_BaseNPC_Hero = context[centaurName];
+            centaur.AddItemByName("item_blink");
+            centaur.AddExperience(30 - centaur.GetLevel(), ModifyXpReason.UNSPECIFIED, false, false);
+            const basicAbilities = [0, 1, 2].map(abilityIndex => getOrError(centaur.GetAbilityByIndex(abilityIndex)));
+            const ultAbility = getOrError(centaur.GetAbilityByIndex(5));
+            basicAbilities.forEach(ab => ab.SetLevel(4));
+            ultAbility.SetLevel(3);
+        }),
+        tg.wait(0.1),
```
Comment (Shushishtok) [inline game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts:76]: Is this wait necessary?

File: game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+            ultAbility.SetLevel(3);
+        }),
+        tg.wait(2),
+        tg.seq([
+            castNoTarget(paName, 2),
+            tg.wait(2),
+            castOnTarget(axeName, paName, 0),
+            tg.wait(0.5),
+            castOnTarget(axeName, paName, 1),
+            tg.wait(6),
```
Comment (Shushishtok) [inline game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts:120]: 6 seconds sounds like a long time to wait without her doing much. Maybe alter the flow a little so there's no such a long downtime?

File: game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+function onStop() {
+    print("Stopping", sectionName);
+
+    if (graph) {
+        graph.stop(GameRules.Addon.context);
+        graph = undefined;
+    }
+}
+
+function castNoTarget(casterName: string, index: number, wait = 0.1) {
```
Comment (Shushishtok) [inline game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts:256]: Those functions are useful and can probably be put as steps. I know we're technically done, but for the other people who might want follow our tutorial methods. Up to you though.

File: game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts (hunk 2 above)
Comment (RobinKa) [reply game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts:120]: That's just waiting for her cooldown, I thought the timing felt fine actually playing it

File: game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts (hunk 3 above)
Comment (RobinKa) [reply game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts:256]: Ye maybe we can replace all of our cast stuff in another PR

File: game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts (hunk 1 above)
Comment (RobinKa) [reply game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts]: removed that

File: game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts (hunk 2 above)
Comment (Shushishtok) [reply game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts:119]: Is she attacking? Couldn't figure that out, if she Phatom Strikes and shows off Coup de Grace in between that's fine.

File: game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts (hunk 3 above)
Comment (Shushishtok) [reply game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts:255]: "casting all his balls" a sentence I never saw coming lol

**Pull request merged**